packages/core/src/storages/request_provider.ts

```diff
@@ -68,6 +71,10 @@ export abstract class RequestProvider implements IStorage {
         eventManager.on(EventType.MIGRATING, async () => {
             this.queuePausedForMigration = true;
         });
+
+        (async () => {
+            this.initialCount = (await client.get())?.totalRequestCount ?? 0;
+        })().catch(() => {});
```

Comment:
why don't we put this into the `open` method instead?

Reply:
that makes much more sense, cheers guys :)